mathlibtools/leanproject.spec

```diff
@@ -0,0 +1,34 @@
+# -*- mode: python ; coding: utf-8 -*-
+
+
+block_cipher = None
+
+
+a = Analysis(['leanproject.py'],
+             pathex=['C:\\Users\\hmonr\\mathlibtools\\mathlib-tools\\mathlibtools'],
```

Comment:
This doesn't look right to me; this path contains your windows username! I don't yet know enough about pyinstaller to suggest what should be here instead though.

Reply:
Based on my testing, it is ok to delete that line "pathex=['C:\\Users\\hmonr\\mathlibtools\\mathlib-tools\\mathlibtools']". 

So what is the next step?